ui/src/dashboards/actions/index.js

```diff
@@ -296,6 +299,19 @@ export const addDashboardCellAsync = dashboard => async dispatch => {
   }
 }
 
+export const cloneDashboardCellAsync = (dashboard, cell) => async dispatch => {
```

Comment:
seems like you could just actually pass the cloned cell to `addDashboardCellAsync`, no?